### DISCUSSION TITLE
Menu for bot

### DISCUSSION THREAD

**Discussion by ignavan39** [go-telegram-bot-api/telegram-bot-api#424] [Q&A] [answered]
is it possible for a bot to make a menu using your lib?

For example : 

![IMG_20210219_154102](https://user-images.githubusercontent.com/68580920/108486964-e66f1f80-72c8-11eb-8460-b46526b0854a.jpg)

**Comment by ignavan39** [comment]
I just didn't find any documentation on this issue

**Comment by temamagic** [comment, marked as answer]
That's not an issue. Wiki here - https://github.com/go-telegram-bot-api/telegram-bot-api/wiki/Simple-Inline-Keyboard

**Reply by suntong** [reply]
Thanks, just it is no longer compiling, I'm getting

```
# github.com/suntong/go_telebot_demos/telegram-bot-api
keyboard.go:35:18: assignment mismatch: 2 variables but bot.GetUpdatesChan returns 1 value
keyboard.go:42:8: bot.AnswerCallbackQuery undefined (type *tgbotapi.BotAPI has no field or method AnswerCallbackQuery)
```

**Reply by suntong** [reply]
@overplumbum, the wiki needs update, and I'm more interested to know how to handle/response to user pressing some buttons. 

thx.
